### PR TITLE
Use components in finders

### PR DESCRIPTION
**Pull request by andysellick**
**This PR must not be merged until [this PR in publishing components](https://github.com/alphagov/govuk_publishing_components/pull/643) is merged and deployed**

Updating the finders to use components for form elements, so they look like this:

screen shot 2018-11-29 at 09 24 46

Examples: 

- https://finder-frontend-pr-684.herokuapp.com/aaib-reports
- https://finder-frontend-pr-684.herokuapp.com/government/policies/all

Trello card: https://trello.com/c/fhhTwfLo/178-use-form-element-components-in-finders

### CODE REVIEW

**Comment by vanitabarrett** [comment]
@andysellick I'm a bit confused by the heroku review app - it seems to always go to the Latest on GOV.UK? 

https://finder-frontend-pr-684.herokuapp.com/aaib-reports
https://finder-frontend-pr-684.herokuapp.com/news-and-communications

**Comment by vanitabarrett** [comment]
@andysellick I'm a bit confused by the review app - the link you posted to aaib-reports doesn't actually go to the aaib finder?

**Comment by andysellick** [comment]
@vanitabarrett yeah I'm confused too - I've tried other links and they all go to that same place. Something's not right, not sure what.

**Comment by sihugh** [comment]
This should be fine for review now.  I've removed the `DEVELOPMENT_FINDER_JSON` environment variable from the review app and it's now working as expected again.  🤦‍♂️

**Comment by sihugh** [comment]
When the text that accompanies a checkbox wraps, the spacing looks a little odd.  I'm not sure I am able to suggest an obvious improvement though.  (There are longer examples than this in the people finder on https://finder-frontend-pr-684.herokuapp.com/news-and-communications)
screenshot 2018-11-30 17 18 45

File: app/views/components/_date-filter.html.erb [hunk 1]
```diff
@@ -7,6 +7,17 @@
...
+      label: {
+        text: name + " after"
+      },
+      name: key + "[from]",
+      id: key + "[from]",
+      value: from_value,
+      controls: aria_controls_id,
+      describedby: "date-help-text-" + key
+    } %>
+<% if false %>
```
Comment (vanitabarrett) [inline app/views/components/_date-filter.html.erb]: Has this been left in by mistake from some testing? There's also one further down (line 47)

Comment (andysellick) [reply app/views/components/_date-filter.html.erb]: D'oh!!!

**Pull request merged**